src/_data/menu/header.yml

```diff
@@ -8,3 +8,5 @@
   permalink: /speakers
 - title: Sponsors
   permalink: /sponsors
+- title: Code of Conduct
+  permalink: /codeofconduct
```

Comment:
Github is complaining about a missing newline here.

<img width="259" alt="Screenshot 2022-12-05 at 13 36 13" src="https://user-images.githubusercontent.com/47985/205565494-05fb60fb-f6ca-4c8b-a514-925c0b163a04.png">

Reply:
Fixed in [97726ea](https://github.com/bangkokrb/rubyconfth/pull/251/commits/97726ea34c524f142850323e73bc985a06c1d426)